index.html

```diff
@@ -34,5 +34,206 @@ <h2 class="title">I'm a software developer</h2>
             <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
         </ul>
     </section>
+
+    <section class="works">
+    
+        <h2 class="title-one">My Recent Works</h2>
+        <hr class="line">
+        <section class="card1">
+            <img src="placeholder.png" alt="big picture">
+            <h3 class="post"> Multi-post stories</h3>
+            <p class="para1">A daily section of privately personalized<br>
+                reads: no account or sign-up are required.<br>
+                has been the industry standard's dummy<br>
+                text ever the 1500s, when an<br>
+                unknown printer took a standard dummy<br>
+                text. 
+            </p>
+            <div class="web">
+                <button class="css">css</button>
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
```

Comment:
- [ ] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") in your works are a list `ul` and each of them a `li`👍